core/examples/luigi-example-js/README.md

```diff
@@ -0,0 +1,34 @@
+# Sample Luigi application written in plain Javascript
+
+## Overview
+
+This is the JS-based sample application which runs with the Luigi framework.
+
+
+## Development
+
+
+To run this application, follow these steps:
+
+
+1. If you do not have Live Server installed, use this command to install it.
+    ```bash
+    npm install -g live-server
+    ```
+    
+2. Install dependencies.
+    ```bash
+    npm install
+    ```
+    
+3. Build modules.
+    ```bash
+      npm run build
```

Comment:
```suggestion
npm run build
```

Reply:
I didn't understand the requested change in this particular one.  If I removed the indentation it breaks the alignment with the other lines.

Reply:
You can see a little bit above, you already solved this. Here it appeared one more time, I don't know why, so that's why it does not make sense anymore. All good. 👍